docs/simple_abci.md

```diff
@@ -27,6 +29,87 @@ The agents are connected to the remote service [DRAND](https://drand.love) throu
   <figcaption>Simple ABCI architecture with four agents</figcaption>
 </figure>
 
+{% include 'requirements.md' %}
+
+## Running the Demo
+
+!!! note
```

Comment:
@DavidMinarsch please check if you are happy with this non-ideal approach.

Reply:
@jmoreira-valory no @dvilelaf should open a PR on top here adding the agent service and then you can finish your PR once merged into this PR